### ISSUE TITLE
Evaluation metrics issues

### ISSUE THREAD

**Issue by Gvinkc** [deepchem/deepchem#1735]
**Description:**
I am getting trouble with getting the scores from the GraphConvModel. I got the results for the metrics recall_score, but not for precison_score ( I got nan) , because of that I am not getting F1score as 'nan'. If we see the formula -> precision = tp/tp+fp and recall = tp/tp+fn. I am wondering which parameter is responsible for getting 'nan' results. Could you please explain how it is happening? I would really appreciate your help.

- DeepChem Version:
- Platform:

```
Your code sample here
```

**Comment by rbharath** [comment]
I don't believe I've seen NaNs from the precision score before... Are you able to get a classification accuracy out? I'm not sure if the model failed to train entirely perhaps.

**Comment by Gvinkc** [comment]
Yeah, it is weird. I have got all the results that can be calculated using deepchem (accuracy score, kappa score, recall  ..). I also tried the tox21 data from deepchem itself using the jupyter notebook provided in deepchem tutorial. I had same issue with precison score and then f1 score. This is the error.

UserWarning: Error calculating metric precision_score: Classification metrics can't handle a mix of binary and continuous-multioutput targets
  warnings.warn("Error calculating metric %s: %s" % (self.name, e))
computed_metrics: [nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan]
{'precision_score': [nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan]}
